Configuration/Template.php

```diff
@@ -47,6 +47,8 @@ class Template extends ConfigurationAnnotation
      */
     private $owner;
 
+    private $inApp;
```

Comment:
I'm not sure about this naming. The real name should be `bundleLess` but it's too long IMHO.